packages/react-router/tsup.config.ts

```diff
@@ -12,6 +12,7 @@ const config = (enableDevWarnings: boolean) =>
     {
       clean: false,
       entry,
+      external: ["react", "react-dom", "react-router"],
```

Comment:
I only added this for the CJS config because the ESM build wasn't bundling `react-router` - but this can be added to both configs is that's safer

Reply:
Even if it's redundant due to the way tsup is treating the ESM build differently to the CJS build, IMO it feels safer and more explicit to define the same externals everywhere.

Also, I think `react` and `react-dom` are redundant in this array. I ran the build locally without these and the `require("react")` and `require("react-dom")` calls were still there. The tsup docs say that `external` is for configuring _additional_ externals, not replacing the default: https://tsup.egoist.dev/#excluding-packages.

Reply:
ahh good catch - I noticed they weren't bundled originally but wasn't entirely sure why so included them out of an abundance of caution.  Good to know that is expected/documented behavior 👍